tests/Resources/Page/ODH/Prometheus/Prometheus.robot

```diff
@@ -214,6 +213,17 @@ Get Target Endpoints Which Have State Up
     ${links}=    Replace String    ${links}    "    ${EMPTY}
     @{links}=    Split String  ${links}  \n
     [Return]    ${links}
+    
+Get Availability Value Change Date 
+    [Documentation]    Returns date of last availability value change
```

Comment:
hmm it looks that it is not returning the date of the last value change, but the date of the last measurement. Is that correct?

The change detection seems to be implemented by these other lines at tests/Tests/900__high_availability/900__openshift/900__openshift.robot
```
49. ${component_downtime_start_date}=    
50.    ...    Wait Until Keyword Succeeds    1m    1s    Get Availability Value Change Date    0
```

Reply:
I've modified the keyword name to clarify its purpose